### PR TITLE
Add note about quotas

### PR DESCRIPTION
**Pull request by choldgraf**
This adds a step to check that quotas are high enough on the infrastructure before an event

### CODE REVIEW

File: docs/topic/cloud-access.md [hunk 1]
```diff
@@ -109,6 +109,14 @@ To do so, follow these steps:
    AWS accounts we create.
 4. Create the account! They'll receive an email with appropriate instructions.
 
+### Create new accounts in our SSO
+
+We often create new accounts in the 2i2c organization if we want to run community-specific infrastructure attached to that account (e.g., a dedicated Kubernetes cluster).
+To create a new Account under our AWS SSO, follow these steps:
+
+1. Do XXX @yuvipanda can you fill in these steps?
```
Comment (choldgraf) [inline docs/topic/cloud-access.md:117]: @yuvipanda could you add in just a few steps here to point people in the right direction?

Comment (choldgraf) [reply docs/topic/cloud-access.md:117]: Are these the instructions we need to follow? https://docs.aws.amazon.com/singlesignon/latest/userguide/step3.html

Comment (choldgraf) [reply docs/topic/cloud-access.md:117]: @yuvipanda can you please confirm yes or no or link to page that has instructions on how to add an account to SSO? We don't need anything complex, just something to spread this knowledge a bit

Comment (choldgraf) [inline docs/topic/cloud-access.md:117]:
```suggestion
```

File: docs/topic/cloud-access.md [hunk 2]
```diff
@@ -109,6 +109,8 @@ To do so, follow these steps:
    AWS accounts we create.
 4. Create the account! They'll receive an email with appropriate instructions.
 
+2. Double check that the **quota** for this account is large enough to withstand expected usage. If not, request a quota via AWS support.
+
```
Comment (choldgraf) [inline docs/topic/cloud-access.md:113]:
```suggestion
```

**Comment by choldgraf** [comment]
OK I don't want to block this PR on adding content on how to add AWS accounts to our SSO, maybe @yuvipanda can take care of that in a follow up (I just don't know how to do it). I've removed those lines from this PR and I think we can just merge it whenever.

**Review by damianavila — APPROVED** [review]
LGTM

**Comment by choldgraf** [comment]
thanks!

**Pull request merged**